stylesheets/blog-style.css

```diff
@@ -0,0 +1,93 @@
+body, html {
+  height: 100%;
+  font-weight: bold;
+  font-size: 1.1em;
+  font-family: 'Rock Salt', cursive;
+}
+
+.background {
+ background-image: url("https://cdn.shopify.com/s/files/1/0908/9770/products/Maria_van_Oosterwyck_-_Bouquet_of_Flowers_in_a_Vase_-_Google_Art_Project.jpg?v=1454857327");
+ height: 100%;
```

Comment:
I've seen a lot of these styles repeated on several css files.  I suggest making **one** css file and sharing the styles, it's more maintainable.  You can add separate files, if you need to add some customization.